src/apps/concepts/components/AddConceptToDictionaryIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+    Fab,
+    Tooltip
+} from "@material-ui/core";
+import {APIConcept} from "../types";
+
+import {Add as AddIcon} from "@material-ui/icons";
+
+
+
+interface AddConceptToDictionaryIconProps {
+    dictionaryToAddTo:string;
+    canAddConcept:string;
+    row:APIConcept|undefined ;
+    addConceptsToDictionary: Function;
+   
+}
+
+
+    export function AddConceptToDictionaryIcon(props: AddConceptToDictionaryIconProps) {
+        const {
+          dictionaryToAddTo,
+          canAddConcept,
+          row,
+          addConceptsToDictionary
+        } = props;
+    return (canAddConcept=="false"  || dictionaryToAddTo==null || dictionaryToAddTo==undefined)? (null):
+        (
+            <Tooltip title="Add concept to Dictionary" data-testid="addConceptToDictionaryIcon">
+                <Fab onClick={() => {
+                                if (addConceptsToDictionary){ 
+                                    addConceptsToDictionary([row]);
+                                    window.history.back() ;
```

Comment:
Why are we changing the page the user is on?

Reply:
`window.history.back() ;`
will definitely direct the user on "Import existing concept from CIEL" (ViewConceptPage)
and it is required in  Issue Discription

Reply:
So, you want think about this flow a little more carefully. What happens when the user lands directly on this page instead of by navigating from the app? This can happen if, for example they copy and paste the link to this page into the browser. Calling `window.history.back()` moves them back in history, but not necessarily where we want them to be in the application. It may be helpful to look at how things like this are handled in the app at present, i.e., when creating a new concept, I can be redirected back to the page containing the concepts in the dictionary.